### DISCUSSION TITLE
SongiApp - web app for songs in SongPro format

### DISCUSSION THREAD

**Discussion by janproch** [SongProOrg/songpro.org#42] [Show and tell]
Hello,

I created SongiApp -  https://songi.app/ . SongiApp is a web application for musicians, it could be used as an e-songbook.
The app is optimized for all device types - mobile phones, tablets, and desktops and for most used modern browsers (Chrome, Safari, Edge, Firefox). It is also optimized for offline usage.

SongiApp uses the SongPro format as the primary format for song editing. All databases are stored on GitHub ([this](https://github.com/songiapp/songidb/blob/main/index.json) is list of "embedded" databases). There is one difference between SongiApp implementation and SongPro format definition, SongiApp stores multiple songs in one text file, songs are delimited with separator line 
`---`

SongApp contains a mechanism for the discovery of other databases in SongPro format stored on GitHub. If GitHub repository follows some rules described [here](https://github.com/songiapp/songiapp), it is automatically offered in SongiApp.

SongiApp is free and open source, all related stuff is on [github](https://github.com/orgs/songiapp/repositories)

It is possible to add SongiApp into "Related Tools" section on [SongPro page](https://songpro.org/) ?
Thanks

**Comment by spilth** [comment]
Sure thing! Do you want to make a PR against https://github.com/SongProOrg/songpro.org/blob/main/source/index.html.md so I know how you want to be listed?

**Comment by janproch** [comment]
Thanks. PR here: https://github.com/SongProOrg/songpro.org/pull/43

**Comment by spilth** [comment]
Merged - thanks for supporting the SongPro format!

**Comment by janproch** [comment]
Thanks. But PR seems not to be merged, only approved - it is still open and there are no changes in main branch of songpro repo https://github.com/SongProOrg/songpro.org/commits/main/

**Comment by spilth** [comment]
Sorry, it's actually merged now.
